Unity-Technologies/ScriptableRenderPipeline
# Properties are now rounded

### Purpose of this PR
This will make the properties on the canvas look the same as the properties in the blackboard.

---
### Testing status
Running Katana
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=sg/ui-ux-properties-rounded&automation-tools_branch=add-platform-filter&unity_branch=trunk

**Manual Tests**: What did you do?
Create a Property in the blackboard. Drag it out on the canvas. It should look similar to what we have in the blackboard. Rounded and only an output.

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Medium

**Halo Effect**: None, Low, Medium, High?
Medium

![rounded_properties](https://user-images.githubusercontent.com/35328557/51605482-7df9bc80-1f0f-11e9-8397-a81b0083166d.gif)
